.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,36 @@
+name: Bug report
```

Comment:
@rarkins can you please review this PR, I've copy/pasted and adjusted stuff from the main Renovate repository. Not sure if that's allowed according to the Renovate main repo license. I think you should have final say on whether we merge this or not. 😉